thoth/python/project.py

```diff
@@ -73,12 +73,20 @@ def from_files(
         without_pipfile_lock: bool = False,
     ):
         """Create project from Pipfile and Pipfile.lock files."""
+        pipfile_path = pipfile_path or "Pipfile"
+        if not os.path.isfile(pipfile_path):
+            raise FileNotFoundError("No Pipfile found in current directory")
```

Comment:
This will raise an error if file is not present. There can be also other errors (io error, perms error). Also, the message is misleading. You were faster with implementation, I opened #252 PTAL :)

Reply:
I see the master here :D Thanks! I will have a look :)